azure-iot-provisioning-devicesdk/azure/iot/provisioning/devicesdk/transport/mqtt_transport.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import uuid
+import logging
+import six.moves.urllib as urllib
+from . import device_provisioning_constant
+from azure.iot.hub.devicesdk.transport.mqtt.mqtt_provider import MQTTProvider
+from .symmetric_key_transport import SymmetricKeyTransport
+
+logger = logging.getLogger(__name__)
+
+
+class MQTTTransport(SymmetricKeyTransport):
+    def __init__(self, provisioning_host, security_provider):
```

Comment:
> __init__( [](start = 7, length = 10)

set self.on_registration_complete to None in the init. While it won't change function, it's a good idea to have a list of all handlers defined in one place for clarity, instead of dynamically adding the attribute from the Registration Client.

Reply:
I realized I totally botched this comment. on_registration_complete is part of the SymmetricKeyRegistrationClient, and it gets set in the sample. 

What I meant to put here was to set on_registration_complete to None in the SymmetricKeyRegistrationClient

---
In reply to: [270205336](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r270205336) [](ancestors = 270205336)